src/peek_data/config/resources/config.yaml.template

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+# @author Sunyoung Yoo
+#
+# Dependencies on other templates:
+
+general:
```

Comment:
If these files are valid yaml, then I would drop the `.template` suffix. It will make it easier, e.g., for text editors to detector the file type automatically. You can name the folder `templates` instead, if you want to.

Reply:
Some of them used to be `not-valid` yaml but now they are so I'll rename them.